### DISCUSSION TITLE
Income tax

### DISCUSSION THREAD

**Discussion by milanogc** [kernc/backtesting.py#331] [Q&A]
Hello! In Brazil it's applied 15% tax on the profits of a trade, in fact it as bit more complicated than that, but... Is there a way of applying 15% tax on a profitable trade? Thanks

**Comment by kernc** [comment]
So far only post festum, by analyzing [trade data in `stats._trades`](https://kernc.github.io/backtesting.py/doc/examples/Quick%20Start%20User%20Guide.html#Trade-data):
```py
stats = bt.run(...)

trades_pl = stats._trades['PnL']
profitable_trades = trades_pl[trades_pl > 0]
profits = profitable_trades.sum()
tax = .15 * profits
...
```

**Reply by milanogc** [reply]
I see, but I would like the profits to be decreased after a trade, since this tax is paid every month. If not, the simulation will reinvest money that it shouldn't have.
